hackathon_site/hardware/serializers.py

```diff
@@ -89,6 +94,20 @@ def get_team_code(obj: Order):
         return obj.team.team_code
 
 
+class IncidentCreateSerializer(IncidentListSerializer):
+    class Meta:
+        model = Incident
+        fields = IncidentListSerializer.Meta.fields
+        read_only_fields = ("team_id",)
+
+    def create(self, validated_data):
```

Comment:
I think once you change this to not subclass the list serializer, the default `create` method will work

Reply:
yep